README.md

```diff
@@ -1 +1,238 @@
-# todo-list-jest
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="img/global_minbar.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>To-Do List Website</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [To-Do List Website] <a name="about-project"></a>
+
+**[To-Do List Website]** This project is to build a To-Do List website.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> This project was built using HTML, CSS and Javascript
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">CSS</a></li>
+    <li><a href="https://www.javascript.com/">Javascript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[ The structure of the website was designed ]**
+- **[webpack is used for the implementation]**
+- **[and and remove items from the to-do list]**
+- **[mark an item as completed for deletion]**
+- **[The javascript code is split into modules]**
+- **[Includes functions testing using jest]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Presentation <a name="Prsentation"></a>
+
+- [Presentation Link]
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://devalibello.github.io/todo-list-jest/dist)
+
```

Comment:
- [OPTIONAL] Your app is yet to be deployed. There's no need to put a broken link as a live demo in your README. It reduces the integrity of your README file. Kindly put `Coming soon or N/A`.

![link-error](https://github.com/devalibello/todo-list-jest/assets/26597432/2c83a6f9-2692-4289-bb44-e44d91a3cc8b)

Reply:
Oops..😮  We forgot to update it from the settings tab. Thanks for pointing out to us 👍 👏